api/controllers/album.js

```diff
@@ -1,13 +1,44 @@
 // function: allAlbums
 // sends all parent albums
-export async function allAlbums(req, res) {
+
+import { query } from 'express';
+
+/**
+ * /albums : all albums
+ * /albums?nameincl={name} : Search for album that matches this name (fuzzy match)
+ * /albums?tag={tag} : Search for albums that have these tags (fuzzy matches)
+ * /albums?keyword={keyword} : Search for albums that have these keywords (fuzzy matches)
+ * /albums?parentid={parentid} : Search for albums that has this albumId as a parent.
+ * /albums?updatedafter={date} : Search for albums that were last updated after this date.
+ */
+export async function albumsBy(req, res) {
   let conn;
+  const nameIncludes = req.query.nameincl;
+  // lint/prettier keeps trying to {tag} but everything breaks then
+  //const { tag } = req.query;
+
+  const tag = req.query.tag;
+  const parentID = parseInt(req.query.parentid, 10);
+  const date = req.query.date;
+  const { keyword } = req.query;
```

Comment:
combine the other props from req.query into 1 destructured statement.

Reply:
what does this mean?